src/index.ts

```diff
@@ -18,6 +18,7 @@ import '@material/mwc-top-app-bar-fixed';
 
 import './components/llama-select-fab';
 import './llaminator.css';
+import * as storage from './storage';
```

Comment:
nit:
```typescript
import { LlamaStorage, mainImageName } from './storage';
```

This is where the `k` prefix would really clarify _what_ it is, as opposed to a free function.

Reply:
done.